int16 centx
int16 centy
int16 sudut
int16 kec_x
int16 kec_y
int16 kec_sudut

